src/targets/maven.ts

```diff
@@ -411,4 +412,14 @@ export class MavenTarget extends BaseTarget {
         )
       : `${moduleName}.jar`;
   }
+
+  private logCmdOutput(output: Buffer | undefined): void {
+    if (output) {
+      if (output.length === 0) {
+        this.logger.debug(`The command didn't have any output.`);
```

Comment:
Hard to know what the command is from this statement, would be helpful to have some details about the command